tests/testHelpers/buildStateTransitionHelpers.ts

```diff
@@ -47,6 +47,7 @@ const EXIT_CODE = '[*]';
 type OmitKnownKeys<TRoomType extends RoomType> = TRoomType extends
   | 'exampleRoom1'
   | 'exampleRoom2'
+  | 'roomceptionRoom'
```

Comment:
Should updating this file be added to https://github.com/normal-devs/klicker-knight#adding-a-new-room-state ?

Reply:
I don't think that is necessary. I made a ticket to dynamically generate OmitKnownKeys from the schemas. Most of our rooms will have more properties than `type` and `playerState`, so this won't need to be updated in those cases. The purpose of this hardcoded list is to stop you from passing bad data to the test setup, but you'd have to intentionally pass bad data where no data is expected. The bad data will still be caught by the data generator.

Reply:
yesss, dynamically generating this would be great